apps/frontend-manage/.env.development

```
@@ -10,3 +10,4 @@ NEXT_PUBLIC_MANAGE_URL="https://manage.klicker.com"
 NEXT_PUBLIC_CONTROL_URL="https://control.klicker.com"
 NEXT_PUBLIC_AUTH_URL="https://auth.klicker.com"
 NEXT_PUBLIC_LTI_URL="https://lti.klicker.com"
+NEXT_PUBLIC_CHAT_URL="https://chat.klicker.com"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)